bytecode/concrete.py

```diff
@@ -506,12 +508,12 @@ def to_concrete_bytecode(self):
         self.varnames.extend(self.bytecode.argnames)
 
         self.concrete_instructions()
-        modified = self.compute_jumps()
-        if modified:
+        for step in range(0, self.max_compute_jumps_steps):
             modified = self.compute_jumps()
-            if modified:
-                raise RuntimeError("compute_jumps() must not modify jumps "
-                                   "at the second iteration")
+            if not modified:
+                break
+        else:
+            raise RuntimeError("compute_jumps() failed to converge")
```

Comment:
It may be interesting to indicate after how many iteration we stopped.

Reply:
good idea; done